marimo/_output/formatters/ipython_formatters.py

```diff
@@ -61,6 +68,9 @@ def display(*objs: Any, **kwargs: Any) -> Optional[DisplayHandle]:
                 # Store the object if display_id is provided
                 if display_id is not None:
                     display_objects[display_id] = output_value
```

Comment:
Cleaning up the wekref doesn't automatically release the object

Reply:
thats fine. them staying in here is harmless.